docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -130,7 +130,3 @@ ${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |' && ${BASH
 ${BASH_ALIASES[openstack]} server list | grep -F '| test | ACTIVE |' && \
   ${BASH_ALIASES[openstack]} server --os-compute-api-version 2.48 show --diagnostics test --fit-width -f json | jq -r '.state' | grep running || echo FAIL
 ----
-
```

Comment:
Ditto this is a revert

Reply:
It's not visible in the "discussion" view on the PR but if you open the file detail it will be visible what it's removing.